src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -348,4 +393,35 @@ public static void addStandardOredictMetal(String suffix) {
             }
         }
     }
+    
+    public static void addStandardOredictGem(String suffix) {
+        NonNullList<ItemStack> gems = OreDictionary.getOres("gem" + suffix);
+        NonNullList<ItemStack> dusts = OreDictionary.getOres("dust" + suffix);
+        NonNullList<ItemStack> ores = OreDictionary.getOres("ore" + suffix);
+        for (ItemStack ore : ores) {
+            RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), StackUtils.size(gems.get(0), 2));
+        }
+
+        if (dusts.size() > 0) {
```

Comment:
I think this block should be inside the check for `ores.size() > 0`?

Reply:
That's right this was just a problem caused by the copy pasting :)

Reply:
it got it's own if now.